Comparison, Wishlist and Translation revision (#26)
ezsystems/commerce-documentation: docs/guide/product_comparison/product_comparison.md

eZ Commerce offers the option to compare products on a single page.
Customers can save products on their own comparison lists.

> Comparison lists are divided per product type (separately for, e.g. sofas, tables, etc.). 

Author: ```suggestion
Comparison lists are divided per kind of product (separately for, e.g. sofas, tables, etc.). 
```

The comparison function is based on the [basket](../basket/basket.md) system.

!!! note

    There can be many comparison lists for all users including unregistered (anonymous) browser sessions.

>     It is recommended to use a service to frequently clean old anonymous comparison lists for sites with heavy traffic.

Reviewer: ```suggestion
    It is recommended that you use a service to frequently clean old anonymous comparison lists for sites with heavy traffic.
```

### Adding a product to a comparison list

Only orderable products can be added to comparison.
If a product is a variant, the compare button is only shown when all characteristics for this variant are selected.

![](../img/product_comparison_1.png "Add to comparison")

> No prices and no product quantity are stored in the comparison list.

Reviewer: ```suggestion
Neither prices nor product quantity details are stored in the comparison list.
```

### Comparison list

The comparison list can be found in the header of the website.

![](../img/product_comparison_2.png)

> The list is divided into categories per kind of product.

Reviewer: ```suggestion
The list is divided into categories by product type.
```

Author: "Product type" can be misleading here, because it is a domain term, and the "kind" here can also refer to a product category

Reviewer: The term "product type" is used in other places above, hence a suggestion to maintain consistency.

Author: Also fixed above: https://github.com/ezsystems/commerce-documentation/pull/26/files#diff-d95a9ad2aafe8c40cd5fab965e5f94c6R5

![](../img/product_comparison_categories.png)

#### Displaying attributes

Attributes (or product specifications) are grouped for display and can be opened and closed by clicking the arrows in their header rows (accordion UI).

##### Collapsing groups and attributes

If all products in a comparison list have the same attribute value, you can configure the comparison's behavior.

###### Show/Hide attributes for the same value

> There is an icon in top left corner to show/hide attributes.

Reviewer: ```suggestion
There is an icon in the top left corner to show/hide the attributes.
```

> If `collapse_attributes` is set to true, hiding/showing the same attributes is enabled.

Reviewer: ```suggestion
If `collapse_attributes` is set to true, hiding/showing attributes with the same value is enabled.
```
